fontforge/python.c

```diff
@@ -4893,11 +4899,10 @@ return( ss );
 	    } else {
 		if ( !c->points[i-1]->on_curve ) {
 		    sp = SplinePointCreate((c->points[i]->x+c->points[i-1]->x)/2,(c->points[i]->y+c->points[i-1]->y)/2);
-		    sp->selected = c->points[i]->selected;
-		    sp->name = copy(c->points[i]->name);
```

Comment:
These two lines were part of the convention of putting the name and selection status of interpolated points on the following off-curve point. If a user wants to set these in the modified interface they should just add an interpolated point to their contour and set them there.